lib/cypress-rails/init.rb

```diff
@@ -24,7 +24,28 @@ def determine_new_config(config_path)
     end
 
     def merge_existing_with_defaults(json_path)
-      Hash[JSON.parse(File.read(json_path)).merge(DEFAULT_CONFIG).sort]
+      existing = JSON.parse(File.read(json_path))
+      base_url = existing["baseUrl"]
```

Comment:
Probably a lot more things to be checked here, but I figured I'd stop at path and scheme just to demonstrate the idea